### ISSUE TITLE
Documentation — Create quality standards and processes for all practice areas

### ISSUE THREAD

**Issue by andreahewitt-odd** [department-of-veterans-affairs/va.gov-team#10359]
### High Level User Story/ies
As a VFS team, I want the standards and process for every practice area created so that I know what success looks like for that practice area. 

As a VSP team member, I want to have a solid artifact that describes the standards and processes for my practice area so VFS teams can easily understand our expectations and I can use it as a tool to guide teams. 

### Problem Statement 
VSP doesn't have clear documentation that explains minimum standards and/or processes for all practice areas. We have the Collaboration Cycle, which guides VFS teams through most of VSP's development requirements, but VFS teams are still missing artifacts and not meeting standards on their own because they don't understand what is required. Some VSP practice areas, including product, are greatly lacking in direction and currently provide no actionable feedback for VFS teams.

How might we define VSP's minimum expectations by practice area for building and launching on VA.gov (and the processes that go along with those standards) so that VFS teams can develop and iterate in an agile while way guaranteeing a good experience for Veterans? 

### Hypothesis or Bet
If we create understandable documentation about VSP's minimum standards by practice area, then VFS teams will require less manual support to build on VA.gov.

### OKR
O2: Teams building products for Veterans meet or exceed VA.gov's standards for security, consistency, and quality.
O2KR1: Security, consistency, and quality standards and processes for all practice areas are published and publicly accessible to teams building on the platform 

### Success metrics 
- Percent of staging reviews that proceed without intervention (without having to reach out with teams for access questions or artifacts that are missing)
- Number of practice areas that have complete documentation of their minimum standards
- Increase in percentage of VSP support responses that include a link to documentation

### We will know we're done when...
- [ ] Each VSP practice area has documented consistent Must/Should/Consider standards for launching
   - [ ] QA
   - [ ] 508
   - [ ] IA
   - [ ] Content
   - [ ] Design 
   - [ ] Product
   - [ ] Analytics
   - [ ] Engineering (will require coordination with engineering teams) 
- [ ] Collaboration cycle process documentation has been audited by documentation experts and high priority edits have been made accordingly

### Artifacts
- Link to product outline to come

### Communication
- Team that owns this super epic: Product Support and Content & IA

**Comment by meganhkelley** [comment]
Convo w/ Joanne, James, Mandi, Andrea 6/25

- Engineering teams have been working on tooling for quality standards — part of performance dashboard (recent Analytics & Insights Design Sprint) was to understand what engineering quality standards should be and how to report those 
- As part of this super epic, teams will need to determine whether engineering platform standards need to be presented in the same way as other practice areas, or whether those should be reported separately on performance dashboard 
- *tl/dr* Mandi and TBD PM should coordinate with Joanne (& engineering team PMs) as this comes together to determine how engineering standards fit in. Discovery work here should include engineering.

Initial discovery questions 
- Identify all existing standards documentation (where is it, what format is it in, etc)

**Comment by joanneesteban** [comment]
### Potential Resources

- [Data Platforms Overview](https://docs.google.com/spreadsheets/d/1gXH7PfI3UCSu8mzEuYqkE5qwSJV6nFO1s8UpDCZm0BA/edit#gid=0): 
  - different sources of data owned by VSP (where some of the engineering quality standards would live)
- [Analytics & Insights Design Sprint Readout](https://docs.google.com/presentation/d/1zX5PRJyY5o5EJ_vPjCrt60J7Xb1Ov-IYHNa-iNbAjog/edit#slide=id.g71e8facf97_0_301)
   - First prototype for Performance Dashboard & alerts (for not meeting quality standards)

**Comment by shiragoodman** [comment]
@andreahewitt-odd can this be closed?
